dandiapi/zarr/models.py

```diff
@@ -88,7 +88,9 @@ def s3_path(self, zarr_path: str) -> str:
 
     def generate_upload_urls(self, path_md5s: list[dict]):
         return [
-            self.storage.generate_presigned_put_object_url(self.s3_path(o['path']), o['base64md5'])
+            self.storage.generate_presigned_put_object_url(
+                self.s3_path(o['path']), o['base64md5'], content_type=o['content_type']
+            )
```

Comment:
while at it, should may be `path_md5s` to be renamed (it is internal, should be ok AFAIK) into e.g. `path_objs` or `path_recs` to signal that it is not just a list (or mapping) of paths to md5s?